### PR TITLE
Ggircs 1085

### PR DESCRIPTION
**Pull request by rajpersram**
Updated the README.md for the CIIP Portal to add more detail and structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,33 @@
-# GGIRCS Industry Portal
 
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=bcgov_cas-ciip-portal&metric=alert_status)](https://sonarcloud.io/dashboard?id=bcgov_cas-ciip-portal)
-[![CircleCI](https://circleci.com/gh/bcgov/cas-ciip-portal.svg?style=svg)](https://circleci.com/gh/bcgov/cas-ciip-portal)
-[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3459/badge)](https://bestpractices.coreinfrastructure.org/projects/3459)
+# Welcome to the CIIP Portal
 
-Readme WIP
+
```
Comment (dleard) [inline README.md:4]: This line here is making the linter fail (extra whitespace)

Comment (kriscooke) [inline README.md:4]: Linter should be fixed if you click "Commit suggestion" here:
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,33 @@
...
+
+## Status
+
+* Quality Gate: [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=bcgov_cas-ciip-portal&metric=alert_status)](https://sonarcloud.io/dashboard?id=bcgov_cas-ciip-portal)
+
+* CircleCI: [![CircleCI](https://circleci.com/gh/bcgov/cas-ciip-portal.svg?style=svg)](https://circleci.com/gh/bcgov/cas-ciip-portal)
+
+* CII Best Practices: [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3459/badge)](https://bestpractices.coreinfrastructure.org/projects/3459)
+
```
Comment (kriscooke) [inline README.md:26]: Just a thought: badges are meant to be quick indicators of quality control status for the dev-interested (who tend not to scroll far past the section of interest - the file tree) on scanning the repo homepage.

Most repos put them in a row at the top of the page, as prominent accountability badges of honor/shame 
eg: [React](https://github.com/facebook/react) [Vue](https://github.com/vuejs/vue) [Angular](https://github.com/angular/angular) 

It shouldn't diminish the readme title/intro if we just kept them in a neat row above the title.

**Comment by kriscooke** [comment]
Thanks for the nice writeup, BTW!

File: README.md (hunk 2 above)
Comment (dleard) [reply README.md:25]: agreed

**Pull request merged**